### ISSUE TITLE
Detects Game Executable as Redistributables

### ISSUE THREAD

**Issue by nalilord** [Codeusa/SteamCleaner#48]
![steamcleaner_2017-01-13_11-27-55](https://cloud.githubusercontent.com/assets/3942265/21926736/569fcaae-d983-11e6-9949-4946d64ade40.png)

At this point I stopped looking what else it found, to risky for me ;)

**Comment by 07416** [comment]
[I've reported the Deathsmiles issue](https://github.com/Codeusa/SteamCleaner/issues/70) — have you noticed this with any other games?

**Comment by nalilord** [comment]
Nope, and to be honest I stopped looking into the tool after this. I was to concerned that it would remove other used or game/software related files.
